src/databricks/labs/ucx/cli.py

```diff
@@ -637,5 +647,13 @@ def lint_local_code(
     linter.lint(prompts, None if path is None else Path(path))
 
 
+@ucx.command
+def export_assessment(w: WorkspaceClient, prompts: Prompts):
```

Comment:
Looks like a duplicate of the above command?

Reply:
Removed now.